packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -217,7 +350,60 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
           if (!result) throw Fail`Failed to delegate.`;
           return tryDecodeResponse(result, MsgDelegateResponse.fromProtoMsg);
         },
+        /**
+         * Only `bondDenom` deposits accepted until #9211, #9063
+         * @param {Payment} payment
+         * @param {{ timeoutTimestamp: bigint }} [opts]
+         * @returns {Promise<{ sequence: number }>}
+         */
+        async deposit(payment, opts) {
+          const brand = await E(payment).getAllegedBrand();
+          const issuer = this.state.brandToIssuer.get(brand);
+          issuer || Fail`Unknown Issuer for Brand ${brand}.`;
+
+          const amount = await E(issuer).getAmountOf(payment);
+          !AmountMath.isEmpty(amount) ||
+            Fail`Payment amount must be greater than 0.`;
 
+          const { localAccount } = this.state;
+          trace('Depositing funds to LCA');
+          // XXX consider adding exposing an interface to withdraw / send
+          // messages from the LCA (e.g., withdraw funds)
+          await E(localAccount).deposit(payment);
+
+          const timeoutTimestamp =
+            opts?.timeoutTimestamp ??
+            (await this.facets.helper.getTimeoutTimestamp());
+
+          trace('Transferring funds to ICA');
+          /**
+           * // TODO can we infer `/ibc.applications.transfer.v1.MsgTransferResponse`?
+           * @type {unknown[]}
+           */
+          const [result] = await E(localAccount).executeTx([
+            typedJson('/ibc.applications.transfer.v1.MsgTransfer', {
+              sourcePort: this.state.transferChannel.sourcePortId,
+              sourceChannel: this.state.transferChannel.sourceChannelId,
+              token: {
+                amount: String(amount.value),
+                // TODO use Amount (of Payment) to determine denom #9211, #9063 (`ibc/toyatom` won't work here)
+                denom: this.state.bondDenom,
+              },
+              sender: this.state.localAccountAddress,
+              receiver: this.state.chainAddress.address,
+              timeoutHeight: {
+                revisionHeight: 0n,
+                revisionNumber: 0n,
+              },
+              // #9324 what's a reasonable timeout? currently using `FIVE_MINUTES_IN_SECONDS`
+              timeoutTimestamp,
+              memo: '',
+            }),
+          ]);
+          trace('MsgTransfer result', result);
+
+          return /** @type {{ sequence: number }} */ (result);
```

Comment:
Was expecting to see a `bigint` or `string` here. Surprised to see a `number`, but this is what I observed with local chains.

Reply:
is that observed data captured in a unit test?

Reply:
It's currently captured in `packages/boot/tools/supports.ts`: https://github.com/Agoric/agoric-sdk/pull/9342/files#diff-5f4f231637e90b9cdfecc29b28c80beb222607ad154bbc266cbf52c6a7c246e9R438-R446


I considered adding a new type helper to `@agoric/cosmic-proto` that's like `JsonSafe`, but wanted to be sure of this behavior before proceeding.  @michaelfig - do you have any insights here? Are `bigint`s cast to `number`s for the `LocalChainAccount` implementation?